admin_scripts/os2display_os/chromium_setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Make Chromium  autostart and set it up with OS2Display.
+
+# Initialize parameters
+
+TIME=$1
```

Comment:
Why is TIME needed?

Reply:
Because AAK wants a delay before the browser is launched to allow the network to get up.